fix cloned peers on django server with sudden frontend quit!!

a third person that did not founded the room does not get their
"data connection with xxx has closed" event, only the founder does, check that out!